api/proto/config/devconfig.proto

```diff
@@ -83,6 +87,9 @@ message EdgeDevConfig {
   // if we set new epoch, EVE sends all info messages to controller
   // it captures when a new controller takes over and needs all the info be resent
   int64 controller_epoch = 25;
+
+  // Active Baseos Config
```

Comment:
Nit: "Activated" aka desired state, not "active" as in currently running state.

Reply:
Hmm.. Thinking more - should we just drop the word "Activated" .. Just BaseOs Config - because there is no more an "Inactive" config.